integrations/flink_connector/flink-connector-timestream/src/test/java/com/amazonaws/samples/connectors/timestream/SinkInitContext.java

```diff
@@ -126,4 +128,19 @@ public void setSinkMetricGroup(SinkWriterMetricGroup sinkMetricGroup) {
     public SinkWriterMetricGroup getSinkMetricGroup() {
         return this.sinkMetricGroup;
     }
-}
\ No newline at end of file
+
+    @Override
+    public <IN> TypeSerializer<IN> createInputSerializer() {
+        return null;
+    }
+
+    @Override
+    public JobID getJobId() {
+        return null;
```

Comment:
It there is a reference documentation stating that JobID can be returned as null and callers would gracefully handle it.

Reply:
This implementation is only to satisfy the interface for testing. The function is never called and serves no purpose. Gracefully handling the null would be up to the caller, but in this case JobID is not in scope for the interface implementation.